extensions/notebook/package.json

```diff
@@ -251,36 +251,41 @@
 			],
 			"standardKernels": [
 				{
-					"name": "Python 3",
-					"connectionProviderIds": []
-				},
-				{
-					"name": "PySpark",
+					"name": "pyspark3kernel",
+					"displayName": "PySpark3",
 					"connectionProviderIds": [
 						"HADOOP_KNOX",
 						"MSSQL"
 					]
 				},
 				{
-					"name": "PySpark3",
+					"name": "pysparkkernel",
+					"displayName": "PySpark",
 					"connectionProviderIds": [
 						"HADOOP_KNOX",
 						"MSSQL"
 					]
 				},
 				{
-					"name": "Spark | R",
+					"name": "sparkkernel",
+					"displayName": "Spark | Scala",
 					"connectionProviderIds": [
 						"HADOOP_KNOX",
 						"MSSQL"
 					]
 				},
 				{
-					"name": "Spark | Scala",
+					"name": "sparkrkernel",
+					"displayName": "Spark | R",
 					"connectionProviderIds": [
 						"HADOOP_KNOX",
 						"MSSQL"
 					]
+				},
+				{
+					"name": "Python3",
```

Comment:
Should python be lower case here?

Reply:
yes. to be consistent with others. change it to lower case.